### PR TITLE
Add `reverse-string`

### PR DESCRIPTION
**Pull request by BNAndras**
This exercise was featured for week 2 of #48in24, but it's pretty straightforward. I skipped the Unicode tests since that is probably better suited for a concept exercise down the line.

### CODE REVIEW

File: exercises/practice/reverse-string/.meta/tests.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+description = "wide characters"
+include = false
+
+[93d7e1b8-f60f-4f3c-9559-4056e10d2ead]
+description = "grapheme cluster with pre-combined form"
+include = false
+
+[1028b2c1-6763-4459-8540-2da47ca512d9]
+description = "grapheme clusters"
+include = false
```
Comment (misicnenad) [inline exercises/practice/reverse-string/.meta/tests.toml:40]:
```suggestion
include = false

```

File: config.json [hunk 2]
```diff
@@ -62,6 +62,14 @@
         "prerequisites": [],
         "difficulty": 1
       },
+      {
+        "slug": "reverse-string",
+        "name": "Reverse String",
+        "uuid": "df9f3e80-acf5-41e0-9321-d4b13b26a036",
+        "practices": [],
```
Comment (misicnenad) [inline config.json:69]:
```suggestion
        "practices": ["strings"],
```

**Review by misicnenad — APPROVED** [review]
Nice and clean

**Comment by misicnenad** [comment]
>  I skipped the Unicode tests since that is probably better suited for a concept exercise down the line.

Cairo supports only ASCII characters, so there's no need to test for anything Unicode-related
